src/main/app/src/components/oppilaitos/Oppilaitos.js

```diff
@@ -77,10 +76,9 @@ const Oppilaitos = (props) => {
           </Typography>
         </Box>
         <Box className={classes.imageContainer} mt={7.5}>
-          <img
-            className={classes.heroImage}
-            src={_.get(oppilaitos, 'oppilaitos.teemakuva') || DefaultHeroImage}
-            alt="Koulutuksen teemakuva"
+          <HeroImage
+            imgUrl={_.get(oppilaitos, 'oppilaitos.teemakuva')}
```

Comment:
Vois varmaan pikkuhiljaa yhdenmukaistaa näitä käyttämänä optional chainingia
```suggestion
            imgUrl={oppilaitos?.oppilaitos?.teemakuva}
```